modules/statics/src/networks.ts

```diff
@@ -629,6 +629,19 @@ class SeiTestnet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://sei.explorers.guru/transaction/';
 }
 
+class Zeta extends Mainnet implements AccountNetwork {
+  name = 'Zeta';
+  family = CoinFamily.ZETA;
+  //  TODO(WIN-142): Zeta is still only in testnet update to mainnet url when it's live
+  explorerUrl = 'https://explorer.zetachain.com/cc/tx/';
+}
+
+class ZetaTestnet extends Testnet implements AccountNetwork {
+  name = 'Testnet Zeta';
+  family = CoinFamily.ZETA;
+  explorerUrl = 'https://athens3.explorer.zetachain.com/cc/tx/';
```

Comment:
getting a `404` on this link

Reply:
@Vijay-Jagannathan 
It will work with the txn hash for example
https://athens3.explorer.zetachain.com/cc/tx/0xd7483b2070db705beb7b16e64a0e5ff048f921f2286697b6fad0c51e1e0c0642

Reply:
just saw that we're following the same pattern above in other coins; naming is off but thats fine